src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1358,13 +1365,14 @@ namespace DataHeatBalance {
     {
         // Members
         bool EnforceZoneMassBalance; // flag to enforce zone air mass conservation
-        bool BalanceMixing;          // flag to allow mixing to be adjusted for zone mass balance
+        int ZoneFlowAdjustment;      // specifies how zone air flow balance is determined (AdjustMixingOnly, AdjustReturnOnly, AdjustMixingThenReturn, AdjustReturnThenMixing, None)
         int InfiltrationTreatment;   // determines how infiltration is treated for zone mass balance
```

Comment:
Added integer member variable that the maps to the different enforcing options. The existing method "BalanceMixing" is replaced with "AdjustMixingOnly".